cmd/snap/cmd_run.go

```diff
@@ -523,6 +523,14 @@ func (x *cmdRun) runCmdUnderStrace(origCmd, env []string) error {
 	go func() {
 		defer func() { filterDone <- true }()
 
+		// do not filter the initial strace output if SNAP_STRACE_SELF
+		// is set, this is useful when tracking down issues with snap
+		// helpers such as snap-confine, snap-exec ...
+		if osutil.GetenvBool("SNAP_STRACE_SELF", false) {
```

Comment:
Instead of a variable name it might be nicer to use the same mechanism that is already being used to convey parameters into strace. For example, how about accepting a _--raw_ option?  We then need to tweak straceOpts to pick it up and return together with the other options:
```
opts, raw := x.straceOpts()
```
The choice of _--raw_ is based on the fact strace does not accept long form options, so that "--" prefix means nothing to it, and thus we can't conflict.